src/mame/mame.lst

```diff
@@ -2204,6 +2204,7 @@ nmaster                         // (c) 1995 Banpresto / Pandorabox
 paccarn                         // (c) 1996 Namco
 paceight                        // (c) 1996 Namco
 pacslot                         // (c) 1996 Namco
+jumbogod                        // (c) 1995 Namco / Toho
```

Comment:
the lists in mame.lst are supposed to be in alphabetical order (some exception has slipped in, but should be avoided).
Can you please put jumbogod in the right order?